### ISSUE TITLE
is_loopback behaviour differs between ipv4 and ipv6 for IPNetwork

### ISSUE THREAD

**Issue by nkeulen** [netaddr/netaddr#222]
IPv4 behaviour:

```python
>>> netaddr.IPAddress('127.0.0.1').is_loopback()
True
>>> netaddr.IPNetwork('127.0.0.1/32').is_loopback()
True
```

IPv6:
```python
>>> netaddr.IPAddress('::1').is_loopback()
True
>>> netaddr.IPNetwork('::1/128').is_loopback()
False
```

I would expect all of the examples above to return True.
But as you can see `netaddr.IPNetwork('::1/128').is_loopback()` doesn't.

Tested on: 0.8.0

**Comment by nkeulen** [comment]
Fixed this in my local version and created a pull request:  https://github.com/netaddr/netaddr/pull/223
If you agree with the fix please merge it :)

**Comment by nkeulen** [comment]
Was fixed with merge of https://github.com/netaddr/netaddr/pull/223

**Comment by jstasiak** [comment]
Released in version 0.9.0 today.
